apprise/plugins/NotifyTwilio.py

```diff
@@ -150,10 +150,16 @@ class NotifyTwilio(NotifyBase):
         'token': {
             'alias_of': 'auth_token',
         },
+        'api_key': {
```

Comment:
Just a minor (OCD) request; can you change this to just `apikey` (no underscore `_`).  Then it's consistent with all of the other plugins.  (sorry to be a pain) :slightly_smiling_face:

Reply:
Changed. No problem, consistency is important 😄